cbx_loco.gemspec

```diff
@@ -0,0 +1,50 @@
+# coding: utf-8
+lib = File.expand_path("../lib", __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require "cbx_loco/version"
+
+Gem::Specification.new do |s|
+  s.name = "cbx_loco"
+  s.version = CbxLoco::VERSION
+  s.authors = ["Cognibox"]
+  s.email = ["developer@cognibox.com"]
+
+  s.summary = %q{Write a short summary, because Rubygems requires one.}
+  s.summary = %q{Provides rake tasks to synchronize translation assets between the codebase and Loco.}
+  s.description = <<-TEXT
+- `rake i18n:extract` extracts assets from server and client code, and uploads them to Loco using the developer API.
+- `rake i18n:import` Imports assets from Loco using developer API into server-specific files and client-specific files
+
+CbxLoco requires configuration of a Loco API key
+TEXT
+  s.post_install_message = %q{Thanks for using CbxLoco! Remember to run rails generate cbx_loco:install}
+
+  # Prevent pushing this gem to RubyGems.org. To allow pushes either set the 'allowed_push_host'
+  # to allow pushing to a single host or delete this section to allow pushing to any host.
+  if s.respond_to?(:metadata)
+    s.metadata["allowed_push_host"] = "TODO: Set to 'http://mygemserver.com'"
```

Comment:
TODO: Set to (something)

Not sure what should be done here, but it looks like at least something should be done...